stix_shifter_modules/secretserver/tests/stix_translation/test_secretserver_stix_to_query.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+from stix_shifter.stix_translation import stix_translation
+import unittest
+
+
+options_file = open('./options.json').read()
```

Comment:
The test is breaking because it's not able to find this file. You can remove this since I don't see this variable being used in your tests. Likewise also remove the options.json file from your tests file if it's not being used.